examples/docker/docker-compose-auxiliary.yml

```diff
@@ -1,4 +1,4 @@
-# This docker-compose file starts supporting services for a wasmCloud ecosystem, including: 
+# This docker-compose file starts supporting services for a wasmCloud ecosystem, including:
```

Comment:
I think we should turn this `auxillary` docker compose into a `docker-compose-otel.yml` file for clarity, and add your collector container that you tested with. Thoughts? I'm fine with this being a follow up, I want it for metrics too

Reply:
Hmm, I'm not sure. This file also sets up the registry container, which doesn't relate to OTEL. And I didn't swap out/include the `otel/opentelemetry-collector` image because AFAIK there's no UI for it. It isn't really useful on its own, beyond local testing to confirm that OTEL data is being collected

Reply:
I think we can approach again once we have metrics in place, I'd be fine with having just a `docker-compose-otel.yml` file. Goal would just be to have some distinct examples to show how you can launch different services alongside wasmcloud. We should have a minimal example too with just NATS.

Reply:
https://github.com/wasmCloud/wasmCloud/pull/1445